util/src/main/scala/geotrellis/util/ResourceReader.scala

```diff
@@ -0,0 +1,10 @@
+package geotrellis.util
+
+import java.io._
+
+object ResourceReader {
+  def asString(filename: String): String = {
```

Comment:
Had talked to Eugene about this.  He suggested putting it in util, but it's not super valuable, so I'll nuke it.